src/websocket-client.ts

```diff
@@ -59,9 +57,8 @@ export class WebsocketClient extends EventEmitter {
       ...options
     };
 
-    this.client = new RestClient(undefined, undefined, this.options.livenet, this.options.restOptions, this.options.requestOptions);
+    this.client = new InverseClient(undefined, undefined, this.options.livenet, this.options.restOptions, this.options.requestOptions);
```

Comment:
Are websockets working with the linear endpoints? I thought this needs to connect to a different url than the inverse stream.

Reply:
Resolved for public, private dev is in progress